app/views/editor/post.html.erb

```diff
@@ -14,9 +14,9 @@
   <% if params[:template] == "question" %>
     <h4 class="visible-sm">Ask a question</h4>
     <div id="research-note-detail" class="hidden-sm">
-      <h4><i class="fa fa-question-circle"></i> Use this form to ask Question about <b><%= params[:tags].split(':').last %></b></h4>
-      <% if DrupalTag.follower_count(params[:tags]) > 0 %>
-        <div class="alert alert-success" role="alert"><%= DrupalTag.follower_count(params[:tags]) %> people have signed up to answer questions on this topic.</div>
+      <h4><i class="fa fa-question-circle"></i> Use this form to ask Question about <b><%= @node.power_tag('question') if @node %></b></h4>
+      <% if @node && DrupalTag.follower_count(@node.power_tag('question')) > 0 %>
```

Comment:
Great call on the `@node` check here. If you ever want a more concise way to write this, you can use `@node.try(:methodname)` -- though I haven't tried it with passing a parameter in -- I think it might be: `@node.try(:power_tag, 'question')` but I'm actually not sure. 
Anyhow just a cool tip I picked up and thought I'd share; `try()` just returns nil if it doesn't find anything, so if `@node` doesn't exist, it doesn't throw the dreaded `Undefined method 'methodname' for nil` error.